rambo/salt_resources/states/python/init.sls

```diff
@@ -1,3 +1,5 @@
+{% set os = salt['grains.get']('os') %}
```

Comment:
I realize this is just a copy of the first line of code in this pull request, so this isn't new, but this line is tripping me up.

1) This should probably just be the simpler `grains['os']`
2) That grain doesn't exist in Rambo. I don't know how the similar extant code in basebox/init.sls is working currently. It's also not set in the Ruby anywhere.
3) Speaking of (a latter change perhaps beyond simply putting the grain in), is there a better way we could grab the OS for Salt?

Reply:
3's got me thinking. This is the third instance of needing a grain set in situ. We're already setting hostname and WEBSITE_BRANCH. I'd like to look for a better solution than the ugliness that https://github.com/terminal-labs/rambo/blob/master/rambo/Vagrantfile#L45-L57 is turning into. Perhaps simply appending to the grain file with an inline bash command. That'll have more data permanence than `salt-call` too.

Reply:
answers for numbers:
1) maybe, I fount the "salt['grains.get']('os')" form a lot on the internet.
2) That grain does exist in Rambo. it is built in. all salt minions have this by default.
3) maybe. but this is the std way.

Reply:
regarding  "I'd like to look for a better solution than the ugliness that https://github.com/terminal-labs/rambo/blob/master/rambo/Vagrantfile#L45-L57 is turning into."  Yes, i want a better way to setup grains. 

I don't want to fix this until we study how to copy around grains in inflation.  The inflation grain operations are inherently more complex.  We might be able to consolidate these into a single grain philosophy.

Reply:
I didn't forgot about default grains. I looked before I commented. Salt docs are hard to find new things in :/